libr/anal/fcn.c

```diff
@@ -124,8 +150,9 @@ static RBNode *_fcn_tree_probe(FcnTreeIter *it, RBNode *x_, ut64 from, ut64 to)
 			x = y;
 			continue;
 		}
-		if (x->addr <= to - 1) {
-			if (from <= x->addr + (x->_size == 0 ? 0 : x->_size - 1)) {
+		set_meta_min_if_needed (x);
```

Comment:
why? it's weird you need to adjust nodes while iterating over them.

Reply:
IMHO this is wrong... if meta.min would ever require to be recomputed, that means the search of the function is already wrong. You can't change the tree when you are in the middle of a traverse.

Reply:
ok removed 31325792d0959b735e913dbf83c7387d0e8844e6 they were added in a copy-and-paste fit, but are harmless